packaging/darwin/Distribution.in

```diff
@@ -21,22 +21,22 @@ function installCheck() {
 	var apps = system.applications.fromIdentifier('com.redhat.codeready.containers');
 	if(apps) {
         my.result.title = 'Update failed';
-        my.result.message = 'CodeReady Containers is running. Please stop the application before updating.';
+        my.result.message = 'Red Hat OpenShift Local is running. Please stop the application before updating.';
```

Comment:
Same file L#3 also needs to be updated, contains `<title>CodeReady Containers</title>`